src/internal/connector/onedrive/item.go

```diff
@@ -105,13 +105,19 @@ func oneDriveItemInfo(di models.DriveItemable, itemSize int64) *details.OneDrive
 		}
 	}
 
+	if di.GetParentReference().GetName() != nil {
```

Comment:
no need for a nil check on `GetParentReference`?

Reply:
Resolved within [06c7045](https://github.com/alcionai/corso/pull/2085/commits/06c70454cd3455c88707011ebf46bf81f753c90d)